rpcs3/Emu/RSX/gcm_enums.cpp

```diff
@@ -563,6 +563,7 @@ enum
 	CELL_GCM_TEXTURE_MAX_ANISO_16 = 7,
 
 	// Texture Filter
+	CELL_GCM_TEXTURE_FILTER_NONE = 0,
```

Comment:
Vertex textures do not specify filtering. In the geforce 7 era, filtering vs textures was not supported at all. I checked this, and on the vertex_texture sample, texture filter modes are set to 0.